packages/core/contracts/financial-templates/expiring-multiparty/PricelessPositionManager.sol

```diff
@@ -699,6 +700,24 @@ contract PricelessPositionManager is FeePayer {
         }
     }
 
+    /**
+     * @notice Accessor method to compute a transformed price identifier using the finanicalProductLibrary specified
+     * at contract deployment. If no library was provided then no modification to the identifier is done.
+     * @param requestTime timestamp the identifier is to be used at.
+     * @return transformedPrice price with the transformation function applied to it.
+     * @dev This method should never revert.
+     */
+    function transformPriceIdentifier(uint256 requestTime) public view returns (bytes32) {
```

Comment:
These should also be nonReentrant b/c they call an external contract?

Reply:
We could add that but this is a view function so don't really see any issues with not having it. 

I'll add it for consistency

Reply:
We guard view functions with `nonReentrantView` all throughout the EMP

Reply:
More than happy to add this. As discussed offline, this will require a small refactor to use an internal method that has no guards & then applying the guards on a public accessor. I will save this for a separate PR. I've created an issue to track it here: https://github.com/UMAprotocol/protocol/issues/2232

Reply:
SGTM